django/db/models/sql/query.py

```diff
@@ -1918,6 +1918,14 @@ def clear_ordering(self, force_empty):
         if force_empty:
             self.default_ordering = False
 
+    def _get_alias_map_tables_column_names(self):
+        joined_models = set()
+        for join in list(self.alias_map.values())[1:]:
+            joined_models.update([join.join_field.model, join.join_field.related_model])
```

Comment:
Do the test pass when only adding `related_model`? That might be the appropriate thing to add.

Reply:
Yes, it passed.